app/features/onboarding/onboarding_controller.rb

```diff
@@ -91,6 +91,8 @@ class OnboardingController < ControllerBase
     # 3) If the data is not valid, display an error message
     post "#{HOME}/ci_bot_account" do
       if valid_params?(params, post_parameter_list_for_ci_bot_user_validation)
+        validate_api_token_correct(params[:ci_user_api_token].strip, "ci_bot_account")
```

Comment:
confusion between `ci_user_api_token` and `ci_bot_account`
maybe `ci_user_api_token` should be `github_bot_api_token`?

Reply:
I like that idea 👍